ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java

```diff
@@ -245,11 +246,6 @@ public void requestBannerAd(Context context,
         }
 
         adLayout = new RelativeLayout(context);
-        // Make adLayout wrapper match the requested ad size, as Vungle's ad uses MATCH_PARENT for

```

Comment:
For smart/adaptive banner/MREC we don't need to let wrapper view match the request ad size. Let's do it on app side.